addons/fullscreen/addon.json

```diff
@@ -0,0 +1,104 @@
+{
+  "name": "Enhanced fullscreen",
+  "description": "Adds several tweaks and customization options to the project player's fullscreen mode, such as syncing with your browser's fullscreen mode (F11), darkening the background behind the project player, and hiding the green flag toolbar.",
+  "credits": [
+    {
+      "name": "lisa_wolfgang",
+      "link": "https://github.com/lisa-wolfgang"
+    }
+  ],
+  "info": [
+    {
+      "type": "notice",
+      "text": "If you hide the toolbar and don't have the fullscreen sync option enabled, you'll need to use the Esc key on your keyboard to exit the project player's fullscreen mode.",
+      "id": "hideToolbarNotice"
+    },
+    {
+      "type": "notice",
+      "text": "If you use the F11 key to enter fullscreen, the Esc key will not fully exit browser fullscreen. If you enable the editor fullscreen option and you're in the editor, when you use F11 and then click the stage button, neither the Esc key nor the stage button will exit browser fullscreen.",
+      "id": "escExitNotice"
+    }
+  ],
+  "settings": [
+    {
+      "name": "Sync browser's and Scratch's fullscreen modes",
+      "id": "browserFullscreen",
+      "type": "boolean",
+      "default": true
+    },
+    {
+      "name": "Only toggle browser fullscreen in editor with stage button",
+      "id": "editorFullscreen",
+      "type": "boolean",
+      "default": true,
+      "if": {
+        "settings": { "browserFullscreen": true }
+      }
+    },
+    {
+      "name": "Darken background",
+      "id": "darkenBg",
+      "type": "boolean",
+      "default": true
+    },
+    {
+      "name": "Hide toolbar",
+      "id": "hideToolbar",
+      "type": "boolean",
+      "default": false
+    }
+  ],
+  "dynamicEnable": true,
+  "dynamicDisable": true,
+  "updateUserstylesOnSettingsChange": true,
+  "userstyles": [
+    {
+      "url": "hideOverflow.css",
+      "matches": ["projects"]
+    },
+    {
+      "url": "removeBorder.css",
+      "matches": ["projects"]
+    },
+    {
+      "url": "resizeWindow.css",
+      "matches": ["projects"],
+      "if": {
+        "settings": {
+          "hideToolbar": false
+        }
+      }
+    },
+    {
+      "url": "resizeWindow_noToolbar.css",
+      "matches": ["projects"],
+      "if": {
+        "settings": {
+          "hideToolbar": true
+        }
+      }
+    },
+    {
+      "url": "darkenBg.css",
+      "matches": ["projects"],
+      "if": {
+        "settings": { "darkenBg": true }
+      }
+    },
+    {
+      "url": "hideToolbar.css",
+      "matches": ["projects"],
+      "if": {
+        "settings": { "hideToolbar": true }
+      }
+    }
+  ],
+  "userscripts": [
+    {
+      "url": "userscript.js",
+      "matches": ["projects"]
+    }
+  ],
+  "tags": ["editor", "theme", "projectPlayer"],
```

Comment:
I don't think we should consider this a theme.
`"featured"` tag would work here too.